packages/launcher/src/widget.tsx

```diff
@@ -174,7 +174,8 @@ export class Launcher extends VDomRenderer<ILauncher.IModel> {
                 icon={icon}
                 iconClass={classes(iconClass, 'jp-Icon-cover')}
                 stylesheet="launcherSection"
-              />
+                aria-hidden="true"
+              />{' '}
```

Comment:
Was this space intentionally added?

```suggestion
              />
```